scripts/install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+os=`echo "$(uname -s)_$(uname -m)"`
+if [ $os == "Linux_x86_64" ] || [ $os == "Linux_armv6" ] || [ $os == "Darwin_x86_64" ] || [ $os == "Linux_arm64" ] || [ $os == "Linux_i386" ]]
+then
+        echo Installing Phoneinfoga.........
+        wget "https://github.com/sundowndev/phoneinfoga/releases/download/v2.3.2/phoneinfoga_$(uname -s)_$(uname -m).tar.gz"
+        tar xfv "phoneinfoga_$(uname -s)_$(uname -m).tar.gz"
+        mv ./phoneinfoga /usr/bin/phoneinfoga
+        echo Installation completed  Successfully.
+else
+        echo Your OS/arch is not Supported.
+fi
+exit 
```

Comment:
Specify the exit code

```diff
-exit
+exit 0
```

Reply:
Thank You so much, I'll make those changes.

Reply:
Thanks 👍